libraries/ESP32/examples/Camera/CameraWebServer/app_httpd.cpp

```diff
@@ -20,6 +20,17 @@
 #include "sdkconfig.h"
 #include "camera_index.h"
 
+//#define CONFIG_LED_ILLUMINATOR_ENABLED 1
+#ifdef CONFIG_LED_ILLUMINATOR_ENABLED
+#define FLASH_LED_GPIO GPIO_NUM_4
+#define CONFIG_LED_MAX_INTENSITY 255
```

Comment:
CONFIG_LED_MAX_INTENSITY - defined but not used anywhere.

Reply:
> CONFIG_LED_MAX_INTENSITY - defined but not used anywhere.

It used for UI led intensity limit
```
#ifdef CONFIG_LED_ILLUMINATOR_ENABLED
void enable_led(bool en)
{ // Turn LED On or Off
    int duty = en ? led_duty : 0;
    if (en && isStreaming && (led_duty > CONFIG_LED_MAX_INTENSITY))
    {
        duty = CONFIG_LED_MAX_INTENSITY;
    }
    ledc_set_duty(CONFIG_LED_LEDC_SPEED_MODE, CONFIG_LED_LEDC_CHANNEL, duty);
    ledc_update_duty(CONFIG_LED_LEDC_SPEED_MODE, CONFIG_LED_LEDC_CHANNEL);
    ESP_LOGI(TAG, "Set LED intensity to %d", duty);
}
#endif
```